packages/hash/api/src/hashGraph/index.ts

```diff
@@ -0,0 +1,29 @@
+import { DataSource } from "apollo-datasource";
```

Comment:
Are we using camelCase for folders? (general question for this module, not this specific line)

Reply:
I appreciate this project has an inconsistent mix 🙃 But I _thought_ we had decided we'd use kebab-case going forwards, and were planning on refactoring the others to be consistent with that.

**_If_** that's the case can we just start using the right one from the start considering it's inconsistent anyway?

Reply:
My bad, yes we should be doing kebab-case

Reply:
Changed to just `graph` a0718d871223a51f060e3d61f95496df8d4bc45c. We can worry about ambiguity with `graphql` later when we touch on that. I think the folder structure as a whole should change